app/views/coronavirus_form/results.html.erb

```diff
@@ -1,4 +1,8 @@
-<% content_for :title do %><%= t('coronavirus_form.results.header.title') %><% end %>
+<% if result_groups(session).empty? %>
+  <% content_for :title do %><%= t('coronavirus_form.results.header.title_no_results') %><% end %>
```

Comment:
@injms pedantic question, do we prefer flow control to be in a helper or we cool with if statements in the view

Reply:
@huwd I will add a commit for this as. I sort of also realised I need to change the title for the meta_tags content below as well... so helper methods will stop the logic in the views building up too much.

Reply:
Added a commit for this.